native_client/dotnet/DeepSpeechClient/DeepSpeech.cs

```diff
@@ -87,40 +87,40 @@ public unsafe int GetModelSampleRate()
         /// Evaluate the result code and will raise an exception if necessary.
         /// </summary>
         /// <param name="resultCode">Native result code.</param>
-        private void EvaluateResultCode(ErrorCodes resultCode)
+        private string EvaluateResultCode(ErrorCodes resultCode)
```

Comment:
I don't think we need to keep that, it was doing the same thing as `DS_ErrorCodeToErrorMessage`, so we can completely get rid of it. We might make it a simple alias if we want for compatibility, but I'd advise agaisnt that since we are not yet 1.0.

@carlfm01 opinion?

Reply:
@lissyx We can't remove `EvaluateResultCode` it is being used to check the result of the native calls, the point of Reuben was to remove the static switch with the error messages on it, so now with the changes, we can do:

```
private void EvaluateResultCode(ErrorCodes resultCode)
{
            if (resultCode != ErrorCodes.DS_ERR_OK)
            {
                throw new ArgumentException(NativeImp.DS_ErrorCodeToErrorMessage(resultCode).IntPtrToString());
            }
}
```

@imskr  And don't return string, it needs to stay void.